src/order_gui.cpp

```diff
@@ -1434,17 +1411,17 @@ struct OrdersWindow : public Window {
 		if (this->vehicle->owner != _local_company) return ES_NOT_HANDLED;
 
 		switch (hotkey) {
-			case OHK_SKIP:           this->OrderClick_Skip();          break;
-			case OHK_DELETE:         this->OrderClick_Delete();        break;
-			case OHK_GOTO:           this->OrderClick_Goto(OPOS_GOTO); break;
-			case OHK_NONSTOP:        this->OrderClick_Nonstop(-1);     break;
-			case OHK_FULLLOAD:       this->OrderClick_FullLoad(-1);    break;
-			case OHK_UNLOAD:         this->OrderClick_Unload(-1);      break;
-			case OHK_NEAREST_DEPOT:  this->OrderClick_NearestDepot();  break;
-			case OHK_ALWAYS_SERVICE: this->OrderClick_Service(-1);     break;
-			case OHK_TRANSFER:       this->OrderHotkey_Transfer();     break;
-			case OHK_NO_UNLOAD:      this->OrderHotkey_NoUnload();     break;
-			case OHK_NO_LOAD:        this->OrderHotkey_NoLoad();       break;
+			case OHK_SKIP:           this->OrderClick_Skip();                   break;
+			case OHK_DELETE:         this->OrderClick_Delete();                 break;
+			case OHK_GOTO:           this->OrderClick_Goto(OPOS_GOTO);          break;
+			case OHK_NONSTOP:        this->OrderClick_Nonstop(-1);              break;
```

Comment:
Just to check, a similar fix is not needed for the other `-1` entries in this enum?

Reply:
"Probably not needed"
* Nonstop uses a "toggle to the next" (its own words) rather than on/off
* Service ... I suppose it could, but there's only 2 options anyway, so nothing noticable